snakemake/utils.py

```diff
@@ -555,3 +556,9 @@ def _find_bash_on_windows():
         except FileNotFoundError:
             bashcmd = ""
     return bashcmd if os.path.exists(bashcmd) else None
+
+
+CondaEnvTuple = namedtuple(
+    "CondaEnvTuple",
+    "is_named env"
+)
```

Comment:
This could also be removed, and instead of using `is_named`, we just using `isinstance(conda_env, str)` in its stead.